### PR TITLE
Add scope doc

### PR DESCRIPTION
**Pull request by jacquesqiao**

### CODE REVIEW

**Comment by shanyi15** [comment]
@tink2123  Please have a format check, thanks

File: paddle/fluid/pybind/pybind.cc [hunk 1]
```diff
@@ -398,7 +398,25 @@ All parameter, weight, gradient are variables in Paddle.
...
+
+    Variables in a parent scope can be retrieved from local scope.
+
+    You need to specify a scope to run a Net, i.e., `exe.Run(&scope)`.
+    One net can run in different scopes and update different variable in the
+    scope.
+
+    You can create var in a scope and get it from the scope.
+
+  For example:
```
Comment (tink2123) [inline paddle/fluid/pybind/pybind.cc]: 为保证预览效果，请老师按以下格式修改代码段

```suggestion
    Examples:
        .. code-block:: python

          # create tensor from a scope and set value to it.
          param = scope.var('Param').get_tensor()
          param_array = np.full((height, row_numel), 5.0).astype("float32")
          param.set(param_array, place)
```
```

Comment (jacquesqiao) [reply paddle/fluid/pybind/pybind.cc]: done

**Review by shanyi15 — APPROVED** [review]
Cool！LGTM

**Pull request merged**